pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -32,15 +34,12 @@ const (
 	// clusterPlatformLabel is the label on a cluster deployment which indicates whether or not a cluster is on GCP platform
 	clusterPlatformLabel = "hive.openshift.io/cluster-platform"
 	clusterPlatformGCP   = "gcp"
-	// TODO(Raf) get name of org parent folder and ensure it exists
-	orgParentFolderID = ""
+	orgParentFolderID    = "240634451310" // Service Delivery org subfolder
```

Comment:
I feel like this should be coming from a config map instead of hard coding

Reply:
+1